test/blocks/media/mocks/body.html

```diff
@@ -72,4 +72,37 @@ <h2 id="använd-acrobat-verktyg-kostnadsfritt">Använd Acrobat-verktyg kostnadsf
     </div>
   </div>
 </div>
+<div class="media subcopy-link">
+  <div class="container foreground">
+    <div>
+      <picture>
+        <source media="(max-width: 400px)" srcset="./">
+        <img alt="" loading="lazy" src="">
+      </picture>
+    </div>
+    <div>
+      <p>Body S 16/24 Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat.</p>
+      <p class="action-area"><strong><a href="https://adobe.com/">Learn More</a></strong></p>
+      <h3 id="watch-tutorial-h3" class="'heading-m', 'body-xl">Watch tutorial H3</h3>
+      <p class="body-s"><strong><a href="https://adobe.com/acrobat"><span class="icon icon-play-circle"></span>Subcopy Link 1</a></strong></p>
+      <p class="body-s"><strong><a href="https://www.adobe.com/acrobat/personal-document-management.html#AdobeTV_341149"><span class="icon icon-play-circle"></span>Subcopy link 2</a></strong></p>
+    </div>
+  </div>
+</div>
+<div class="media qr-code">
```

Comment:
Same w/ this. `qr-code` gets added via the block `.js`
The mock/body.html pages should represent the `.plain.html` views when the DOM is first rendered so that we can validate the functions are doing what they're suppose too.

Reply:
Hi @ryanmparrish , the qr-code media block has been created as a media block's variant - therefore "qr-code" gets added right away to classList for this div

Reply:
Ah sorry, I overlooked that. Sounds good.